DataFormats/L1TParticleFlow/interface/PFTau.h

```diff
@@ -49,6 +51,7 @@ namespace l1t {
 
     float z0() const { return vz(); }
     float dxy() const { return dxy_; }
+    const float* NNValues() const { return NNValues_; }
 
     bool passMass() const { return (mass() < 2 + pt() / PTSCALING_MASSCUT); }
     bool passLooseNN() const {
```

Comment:
Does this function now return the pass/fail decision based on your new working point?